packages/react-components/src/components/RTE/RTESendBox.tsx

```diff
@@ -53,6 +59,10 @@ export interface RTESendBoxProps {
    * @beta
    */
   onCancelFileUpload?: (fileId: string) => void;
+  /**
+   * Optional override behavior on send button click
```

Comment:
The comment here implies that the message gets sent regardless of whether this is set(implemented).
Is this callback really used to implement the send button functionality?

Reply:
I'm not sure that I agree.
The comment specify that this callback will change the behaviour for a send button click event, the default behavious is that clicks don't cause any action for sending. 

Just wanted to add that the same pattern is reused from SendBox and I see that we've also used the same pattern for other callbacks, like cancelling file uploads. And some React/HTML events that don't have any default behavior.  

But I'm open for any suggestions for the name of this callback

Reply:
My understanding is that the callback is called when we want to send the message contents.
If this is the case, I'd like to see the description be more like:

`Function called when the send message button is pressed`

If the original SendBox also has this description, maybe that too should be updated?

Reply:
Made the callback required and updated the comment as agreed